src/main/java/esa/sen2vm/utils/LevelInfo.java

```diff
@@ -0,0 +1,72 @@
+package esa.sen2vm;
+
+/**
+ * Information on detector
+ */
+public enum LevelInfo {
```

Comment:
Do we need this, we will only get L1B.
But indeed, we could think about extend it to L1A as model are very similar.

Reply:
Okay I remove this enum, we can put it back when we will actually need it